librz/bin/p/bin_pe.inc

```diff
@@ -249,8 +261,15 @@ static RzList *symbols(RzBinFile *bf) {
 			ptr->bind = RZ_BIN_BIND_GLOBAL_STR;
 			ptr->type = RZ_BIN_TYPE_FUNC_STR;
 			ptr->size = 0;
-			ptr->vaddr = symbols[i].vaddr;
-			ptr->paddr = symbols[i].paddr;
+			if (is_thumb(bin, symbols[i].vaddr)) {
+				ptr->bits = 16;
+				ptr->vaddr = symbols[i].vaddr - 1;
+				ptr->paddr = symbols[i].paddr - 1;
```

Comment:
why -1 ?

Reply:
I think it's because it's thumb and thus the address you get is +1.